storage/src/main/java/org/apache/kafka/storage/internals/log/ProducerStateManagerConfig.java

```diff
@@ -24,8 +24,11 @@ public class ProducerStateManagerConfig {
     public static final Set<String> RECONFIGURABLE_CONFIGS = Collections.singleton(PRODUCER_ID_EXPIRATION_MS);
     private volatile int producerIdExpirationMs;
 
-    public ProducerStateManagerConfig(int producerIdExpirationMs) {
+    private volatile boolean transactionVerificationEnabled;
```

Comment:
small nit: I would put an empty line before `producerIdExpirationMs` and remove the one between `producerIdExpirationMs` and `transactionVerificationEnabled`.

Reply:
can do